src/test/store/receive-profile.test.js

```diff
@@ -1142,12 +1099,11 @@ describe('actions/receive-profile', function () {
 
     it('can retrieve a gzipped profile from the web and save it to state', async function () {
       const expectedUrl = 'https://profiles.club/shared.json';
-      window.fetch = jest.fn((url) =>
-        Promise.resolve(
-          url === expectedUrl ? fetch200GzippedResponse : fetch403Response
-        )
+      window.fetch.get(
+        expectedUrl,
+        compress(serializeProfile(_getSimpleProfile())),
+        { sendAsJson: false }
```

Comment:
`sendAsJson: true` is the default behavior. This means that if we send something that's not a normal object (like array buffers here), we need to specify `sendAsJson: false`. We might want to do the opposite: set `sendAsJson` false by default, but set it to `true` when needed. This might be more intuitive.

I decided to wait a bit before changing the default, to see which one would be better.

Reply:
Hm yeah, I don't feel strongly about either solution. Waiting and seeing works for me.